src/data/real/ereal.lean

```diff
@@ -121,6 +125,38 @@ protected def mul : ereal → ereal → ereal
 
 instance : has_mul ereal := ⟨ereal.mul⟩
 
+/-- Induct on two ereals by performing case splits on the sign of one whenever the other is
+infinite. -/
+@[elab_as_eliminator]
+lemma induction₂ {P : ereal → ereal → Prop}
```

Comment:
A priori there are many possible induction principles on two ereals. Should we add `sign` or `bot`/`top` somewhere in the name?

Reply:
`bot`/`top` is implied by `ereal.rec` so I don't think that's helpful. `sign` is maybe ok, but the goals don't actually mention `_root_.sign`. I think it's ok to give it this name, and let someone rename it if they need a competing principle.